homeassistant/components/google_generative_ai_conversation/__init__.py

```diff
@@ -91,8 +95,42 @@ def append_files_to_prompt():
                     )
                     prompt_parts.append(uploaded_file)
 
+        async def wait_for_file_processing(uploaded_file: File) -> None:
+            """Wait for file processing to complete."""
+            uploaded_file = client.files.get(
+                name=uploaded_file.name,
+                config={"http_options": {"timeout": TIMEOUT_MILLIS}},
+            )
+            while uploaded_file.state in (
+                FileState.STATE_UNSPECIFIED,
+                FileState.PROCESSING,
+            ):
+                await asyncio.sleep(FILE_POLLING_INTERVAL_SECONDS)
+                LOGGER.debug(
+                    "Waiting for file `%s` to be processed, current state: %s",
```

Comment:
Move the log message before the sleep

Reply:
Done

Reply:
Now this is logged even if there is no wait. Move it right before the sleep

Reply:
Done now